ci/docker/ml.build.Dockerfile

```diff
@@ -34,7 +34,7 @@ fi
 if [[ "$RAYCI_LIGHTNING_2" == "true" ]]; then
   pip uninstall -y pytorch-lightning
   # todo move to requirements-test.txt
-  pip install lightning==2.0.4 pytorch-lightning==2.0.4  
+  pip install lightning==2.1.2 pytorch-lightning==2.1.2
```

Comment:
Let's update the [docs](https://github.com/ray-project/ray/blame/6605a4b798ba7ed97c3b8702f61ea6e03752e9f5/doc/source/train/getting-started-pytorch-lightning.rst#L390-L392) as well?